release/release-dockerhub

```diff
@@ -29,13 +29,14 @@ SRPM=${RELEASE_SRPM:-$PWD/srpm}
 CHECK=${RELEASE_CHECK:-0}
 
 REPO=""
+SOURCE_REPO=""
 WORKDIR=""
 PUSH=""
 CLEANUP=""
 
 usage()
 {
-    echo "usage: release-dockerhub [-qvxz] [-p SRPM] REPO" >&2
+    echo "usage: release-dockerhub [-qvxz] [-p SRPM] [-s SOURCE_REPO ] REPO" >&2
```

Comment:
We always need a target repo. Having two ways to pass it in seems more confusing to me.

Or did you mean add a -r REPO required argument and remove the $1 REPO argument?